### PR TITLE
Add February 2022 news 

### PR DESCRIPTION
**Pull request by cticenhour**
@idaholab/moose-ccb Thank you all for making sure news entries are being added with major PRs!  If there is anything that has been missed, please push it to this branch, or get in touch with me about adding more content. Thanks! 

Refs #11447

### CODE REVIEW

**Comment by loganharbour** [comment]
We should probably add another note about how legacy parameter construction is being removed

**Comment by GiudGiud** [comment]
This is also missing NS rework and some misc things. i ll make a PR later today

**Comment by cticenhour** [comment]
@GiudGiud if you want to push to my branch for that you can do so directly. Should have rights as a member of the CCB.

**Comment by GiudGiud** [comment]
Added something about NS.
@lindsayad if you want to review it

File: modules/doc/content/newsletter/2022_02.md [hunk 1]
```diff
@@ -126,10 +121,27 @@ as well as provide excess torque to be applied to a generator for the production
...
+interpolations of velocity. The user object gathers all the contributions to the diagonal
+term in the momentum equation, allowing for performing Rhie Chow interpolation on the
+face with the maximum amount of information about the contributions of the kernels (diffusion,
+ friction term, ...) to the momentum equations.
+
+This allows other objects to query face-interpolated velocities, from the user object, facilitating
+future development of boundary conditions, postprocessors, auxiliary kernels and others.
+
+The [ReynoldsNumberFunctorAux.md] auxliary kernels and [RayleighNumber.md] postprocessor were
+to help modelers understand characteristics of the flow they are studying, guiding their
```
Comment (lindsayad) [inline modules/doc/content/newsletter/2022_02.md:137]:
```suggestion
added to help modelers understand characteristics of the flow they are studying, guiding their
```

**Review by lindsayad — COMMENTED** [review]
@GiudGiud overall I think the NS contributions look good!

File: modules/doc/content/newsletter/2022_01.md [hunk 2]
```diff
@@ -62,6 +62,14 @@ generated by this mesh generator.
       id=quad_peripheral_mesh
       caption=An example peripheral ring (cyan region) generated by `PeripheralRingMeshGenerator`.
 
+### Flux boundary conditions for the Navier Stokes finite volume implementation
+
+Mass, momentum and energy flux boundary conditions were added in order to facilitate
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2022_01.md:67]: Could you name these with links to their documentation?

File: modules/doc/content/newsletter/2022_01.md [hunk 3]
```diff
@@ -86,3 +94,6 @@ generated by this mesh generator.
 - A boolean parameter `extrap` was added to [PiecewiseLinear.md], which when set
   to `true` specifies that extrapolation occurs when the sample is outside of the
   provided abscissa values, instead of using the nearest ordinate value.
+- Documentation was added for certain [Problems](syntax/Problem/index.md) and [Predictors](syntax/Executioner/Predictor/index.md)
+- Additional utilities for the functor system were added, such as to create functor
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2022_01.md:98]:
```suggestion
- Additional utilities for the functor system were added, such as to create a functor
```

File: modules/doc/content/newsletter/2022_02.md [hunk 4]
```diff
@@ -126,10 +121,27 @@ as well as provide excess torque to be applied to a generator for the production
 of electricity. For more information on these examples, see
 [thermal_hydraulics/modeling_guide/brayton_cycle/brayton_cycle.md].
 
-## libMesh-level Changes
+### Finite volume Navier Stokes incompressible and weakly compressible implementation
+
+The finite volume discretization of the Navier Stokes incompressible and weakly compressible
+equations was reworked to feature a [INSFVRhieChowUserObject.md], which handles face
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2022_02.md:127]:
```suggestion
equations was reworked to feature a [INSFVRhieChowInterpolator.md], which handles face
```
Is this the page you meant to link?

File: modules/doc/content/newsletter/2022_01.md (hunk 2 above)
Comment (GiudGiud) [reply modules/doc/content/newsletter/2022_01.md:67]: will do, adding a commit in next 30 mins at most

Comment (GiudGiud) [reply modules/doc/content/newsletter/2022_01.md:67]: Force pushing btw
I ll pull right before just in case

Comment (cticenhour) [reply modules/doc/content/newsletter/2022_01.md]: Looks good - thank you!

File: modules/doc/content/newsletter/2022_01.md [hunk 5]
```diff
@@ -62,6 +62,15 @@ generated by this mesh generator.
       id=quad_peripheral_mesh
       caption=An example peripheral ring (cyan region) generated by `PeripheralRingMeshGenerator`.
 
+### Flux boundary conditions for the Navier Stokes finite volume implementation
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2022_01.md:65]: @GiudGiud I don't know how I missed this earlier, but did you mean to put this and your bulleted additions below into the January news?

File: modules/doc/content/newsletter/2022_02.md [hunk 6]
```diff
@@ -1,9 +1,13 @@
...
-for a complete description of all MOOSE changes.
-!alert-end!
+## Legacy Input Parameter Removal
+
+As announced in November 2021, the legacy input parameter construction method
+(`InputParameters validParams<T>()`) is being removed on 1 March 2022 after two
+years of support. Any remaining templated methods will not compile after this date.
+For information on how to convert source code to the current input parameter
+construction method, directions can be found in the
+[original deprecation announcement](2021_11.md#legacy-input-param-deprecation).
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2022_02.md:10]: @loganharbour I added a legacy input parameter removal notice - if you think we need something more expansive, let me know. I added a link to the fixup directions for those who still need it.

Comment (loganharbour) [reply modules/doc/content/newsletter/2022_02.md:10]: Nope, this is perfect 👍🏼

File: modules/doc/content/newsletter/2022_01.md (hunk 5 above)
Comment (GiudGiud) [reply modules/doc/content/newsletter/2022_01.md:65]: well it technically happened in January

Comment (cticenhour) [reply modules/doc/content/newsletter/2022_01.md:65]: If it happened in January, that is fine. Just wanted to double-check.

**Review by loganharbour — APPROVED** [review]

**Pull request merged**